pysierraecg/src/sierraecg/lib.py

```diff
@@ -107,7 +123,38 @@ def assert_leads(elt: Document) -> List[EcgLead]:
         lead.samples = waveform_data[index]
         leads.append(lead)
 
-    return leads
+    return leads, labels
+    
+def assert_reps(elt, labels):
+    
+    # get repbeats dataencoding
+    rep_encoding = get_attr(get_node(elt, "repbeats"), "dataencoding")
+    rep_sampleingrate = get_attr(get_node(elt, "repbeats"), "samplespersec")
+    rep_resolution = get_attr(get_node(elt, "repbeats"), "resolution")
+    rep_method = get_attr(get_node(elt, "repbeats"), "repbeatmethod")
+    # not using get_nodes becuase cast only returns one waveform
+    rep_repbeats = elt.getElementsByTagName("repbeat")
+    rep_waveforms = elt.getElementsByTagName("waveform")
+    # check leads in same order as parsedwaveform
+    rep_labels = [get_attr(item, "leadname") for item in rep_repbeats]
+    if rep_labels != labels:
+        raise ValueError("Ordering of Leads not same for parsedwaveform and repbeats")
```

Comment:
XML is funny, as I read the XSD I don't think these elements _have_ to be in the same order. I think the more idiomatic way of doing this would be either:
1. Using a `dict` as the order isn't exactly important
2. If order matters, re-ordering based on `leads`

Another option is _attaching_ the rep beat to its respective lead.

Reply:
Order doesn't matter to me, so I think the dictionary way would work great! Do you think any of these changes in this pull request are something you are going to want to incorporate into your main repo? I'm happy to keep working in my forked repo, but if you were considering incorporating any of these changes, I'm happy to defer to you the best way to make this change.